src/engine/specification.md

```diff
@@ -307,6 +319,27 @@ The payload build process is specified as follows:
 
 1. Given the `payloadId` client software **MUST** return the most recent version of the payload that is available in the corresponding build process at the time of receiving the call.
 
-1. The call **MUST** return `-32001: Unknown payload` error if the build process identified by the `payloadId` does not exist.
+2. The call **MUST** return `-32001: Unknown payload` error if the build process identified by the `payloadId` does not exist.
+
+3. Client software **MAY** stop the corresponding build process after serving this call.
+
+### engine_exchangeTransitionConfigurationV1
+
+#### Request
+
+* method: `engine_exchangeTransitionConfigurationV1`
+* params:
+  1. `transitionConfiguration`: `Object` - instance of [`TransitionConfigurationV1`](#TransitionConfigurationV1); `terminalBlockNumber` **MUST** be set to `0`
```

Comment:
Why must `terminalBlockNumber` be set to 0?

couldn't the config be overridden to use TBH and TBN?

Reply:
TBN is always 0 on CL. Just TBH is used